### PR TITLE
Update link README.md

### PR DESCRIPTION
**Pull request by sguequierre**
Fix broken link + update instructions to reflect new template

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,14 +15,14 @@ On the new component panel, copy and paste the following attribute template into
...
-> [!NOTE]  
+> [!NOTE]
 > For more information, see [Configure a Machine](https://docs.viam.com/build/configure/).
 
 ### Attributes
 
 The following attributes are available for `viam:base:agilex-limo` bases:
 
-| Name | Type | Inclusion | Description |
+| Description |
```
Comment (sguequierre) [inline README.md]:
```suggestion
| Name | Type | Inclusion | Description |
```

File: README.md [hunk 2]
```diff
@@ -1,18 +1,12 @@
-# [`agilex-limo` module](https://app.viam.com/module/viam/agilex-limo)
-
-This module implements the [`rdk:component:base` API](https://docs.viam.com/components/base/#api) in an `agilex` model for the [AgileX LIMO](https://global.agilex.ai/education/4) base to be used with [`viam-server`](https://docs.viam.com/). This driver supports differential, ackermann, and omni directional steering modes over the serial port.
```
Comment (andf-viam) [inline README.md:3]: This section [remains](https://github.com/viamrobotics/docs/blob/b2ded2e8193053a4a714500d6ac652ee084e0a94/docs/registry/create/_index.md?plain=1#L1167-L1169) in the template, should it remain here?

File: README.md [hunk 3]
```diff
@@ -50,9 +44,9 @@ The following attributes are available for `viam:base:agilex-limo` bases:
 
 ## Next steps
 
-- To test your base, go to the [**Control** tab](https://docs.viam.com/fleet/machines/#control).
-- To write code against your base, use one of the [available SDKs](https://docs.viam.com/program/).
-- To view examples using a base component, explore [these tutorials](https://docs.viam.com/tutorials/).
+- To test your base, go to the [**CONTROL** tab](https://docs.viam.com/fleet/control/).
+- To write code against your base, use one of the [available SDKs](https://docs.viam.com/build/program/).
+- To view examples using a base component, explore [these tutorials](https:/docs.viam.com/tutorials/).
```
Comment (andf-viam) [inline README.md]:
```suggestion
- To view examples using a base component, explore [these tutorials](https://docs.viam.com/tutorials/).
```

**Review by andf-viam — APPROVED** [review]
LGTM mod one link fix and one question. Thank you!

File: README.md (hunk 2 above)
Comment (sguequierre) [reply README.md:3]: Oh yeah sorry oops!

File: README.md [hunk 4]
```diff
@@ -2,17 +2,15 @@
 
 This module implements the [`rdk:component:base` API](https://docs.viam.com/components/base/#api) in an `agilex` model for the [AgileX LIMO](https://global.agilex.ai/education/4) base to be used with [`viam-server`](https://docs.viam.com/). This driver supports differential, ackermann, and omni directional steering modes over the serial port.
 
-> [!NOTE]
-> For more information, see [Modular Resources](https://docs.viam.com/registry/#modular-resources).
-
-## Configure your `agilex-limo` base
+# Configure your `agilex-limo` base
```
Comment (sguequierre) [inline README.md]:
```suggestion
## Configure your `agilex-limo` base
```

**Pull request merged**